### PR TITLE
Add default option for Mac Os to prevent stealing focus

### PR DESCRIPTION
**Pull request by lcd1232**
Also I removed regex because there is no reason to use it

### CODE REVIEW

File: tabula/wrapper.py [hunk 1]
```diff
@@ -11,12 +11,14 @@
...
 import os
+import platform
 import shlex
 import subprocess
-import requests
-import re
-import pandas as pd
+
 import numpy as np
+import pandas as pd
```
Comment (chezou) [inline tabula/wrapper.py]: Is there any reason to reorder import?

Comment (lcd1232) [reply tabula/wrapper.py]: I just used 'optimize imports' in PyCharm. It reorders imports for PEP8. You can read more about it [here](https://www.python.org/dev/peps/pep-0008/#imports)

Comment (lcd1232) [reply tabula/wrapper.py]: Also [here](https://stackoverflow.com/a/20763446) is a good answer how to works with import

Comment (chezou) [reply tabula/wrapper.py]: Thanks for pointing it out!

**Comment by chezou** [comment]
Will add this behavior to FAQ.

**Pull request merged**